bip-musig2/reference.py

```diff
@@ -708,9 +801,10 @@ def test_sign_and_verify_random(iters):
         # Wrong message
         assert not partial_sig_verify(psig_1, pubnonces, pubkeys, tweaks, is_xonly, secrets.token_bytes(32), 0)
 
-        psig_2 = sign(secnonce_2, sk_2, session_ctx)
-        # Clear the secnonce after use
-        secnonce_2 = bytes(64)
+        if i % 2 == 0:
+            psig_2 = sign(secnonce_2, sk_2, session_ctx)
+            # Clear the secnonce after use
+            secnonce_2 = bytes(64)
```

Comment:
Does this really work? I can't imagine that this is reliable in a language like Python, and I don't think we should give readers the impression that this works. (But let's resolve this in another PR maybe. I see it's consistent with some other comments that we already have somewhere else in the reference code.)